test/stdlib/DispatchDeprecationWatchOS.swift

```diff
@@ -1,5 +1,5 @@
 // RUN: %swift -typecheck -target i386-apple-watchos2.0 -verify -sdk %sdk %s
-// REQUIRES: OS=watchos
+// REQUIRES: CPU=x86_64, OS=watchos
```

Comment:
We can use `\s*,\s*` as a separator
https://github.com/apple/swift-llvm/blob/stable/utils/lit/lit/TestRunner.py#L841